Sources/Components/TJTPriceView/TJTPriceViewModel.swift

```diff
@@ -0,0 +1,79 @@
+import FinniversKit
+import Foundation
+
+public struct TJTPriceViewModel {
+    public struct Shipping {
```

Comment:
All the internal classes / structs should be placed in an extension at the end of the file. 😉